README.md

```diff
@@ -1,7 +1,6 @@
-# Code Climate CLI
+# Wolox Code Climate CLI
 
-[![Code Climate](https://codeclimate.com/github/codeclimate/codeclimate/badges/gpa.svg)](https://codeclimate.com/github/codeclimate/codeclimate)
-[![CircleCI](https://circleci.com/gh/codeclimate/codeclimate.svg?style=svg)](https://circleci.com/gh/codeclimate/codeclimate)
+This is a fork from the [Codeclimate CLI](https://github.com/wolox/codeclimate).
```

Comment:
Is this url okey? Shouldn't it be `https://github.com/codeclimate/codeclimate`?

Reply:
yes, my bad